index.js

```diff
@@ -1,14 +1,31 @@
-'use strict'
+'use strict';
 
-const {BaseKonnector} = require('cozy-konnector-libs')
+const { BaseKonnector } = require('cozy-konnector-libs');
+
+const konnectorErrors = [
+  'CHALLENGE_ASKED',
+  'LOGIN_FAILED',
+  'LOGIN_FAILED.NEEDS_SECRET',
+  'LOGIN_FAILED.TOO_MANY_ATTEMPTS',
+  'MAINTENANCE',
+  'NOT_EXISTING_DIRECTORY',
+  'USER_ACTION_NEEDED.ACCOUNT_REMOVED',
+  'USER_ACTION_NEEDED.CHANGE_PASSWORD',
+  'VENDOR_DOWN.BANK_DOWN',
+  'VENDOR_DOWN.LINXO_DOWN'
+];
 
 module.exports = new BaseKonnector(fields => {
-  const timeout = Number(fields.timeout) || 8000
+  if (!!fields.login && konnectorErrors.includes(fields.login.toUpperCase())) {
+    throw new Error(fields.login.toUpperCase())
```

Comment:
I think it would be better to throw this error after the timeout

Reply:
Is there any specific reason ? Using Debug konnector frequently I am pretty bored by waiting every time :D

Reply:
Then you can fill the timeout field... But since you will be the main user of this connector, I am Ok with it.

Reply:
Ah ok I see, I did not see that this field was here. Actually it is not defined in Collect's `konnectors.json`. But we can fix it.

Reply:
Ok, I did not check the konnectors.json, I understand.

Reply:
I've just added a dedicated commit.

Reply:
@doubleface https://github.com/cozy/cozy-collect/pull/603